pkg/util/atlastest/project.go

```diff
@@ -0,0 +1,103 @@
+package atlastest
+
+import (
+	"context"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type ProjectsServiceMock struct {
+	GetAllProjectsFn func(context.Context, *mongodbatlas.ListOptions) (*mongodbatlas.Projects, *mongodbatlas.Response, error)
```

Comment:
The mocked func doesn't need to have the same signature as the interface, we usually don't care about context in our mock, for example.

```suggestion
	GetAllProjectsFn func() (*mongodbatlas.Projects, *mongodbatlas.Response, error)
```

Reply:
Right!
I will simplify that, makes sense.

Reply:
coming on next change